packages/consent-manager/index.test.js

```diff
@@ -49,42 +49,19 @@ const defaultProps = {
     },
   ],
   container: '#consent-manager',
-  forceShow: true,
 }
 
 test('shows the banner if the forceShow prop is true', () => {
   render(<ConsentManager {...defaultProps} forceShow={true} />)
   expect(screen.getByTestId('consent-banner')).toBeInTheDocument()
 })
 
-test('shows the banner if in EU', async () => {
-  await runWithMockedImport(
-    '@segment/in-eu',
-    () => true,
-    (MockedConsentManager) => {
-      render(<MockedConsentManager {...defaultProps} forceShow={false} />)
-      expect(screen.queryByTestId('consent-banner')).toBeInTheDocument()
-    }
-  )
-})
-
-test('does not show the banner if not in EU', async () => {
-  await runWithMockedImport(
-    '@segment/in-eu',
-    () => false,
-    (MockedConsentManager) => {
-      render(<MockedConsentManager {...defaultProps} forceShow={false} />)
-      expect(screen.queryByTestId('consent-banner')).not.toBeInTheDocument()
-    }
-  )
-})
-
 test('sets existing preferences and does not show the banner if preferences are already set', async () => {
   await runWithMockedImport(
     './partials/cookies',
     {
       loadPreferences: () => {
-        return { All: false, 'Segment.io': false }
+        return { loadAll: false, segment: false, version: 0 }
```

Comment:
The missing version was causing this test to fail, and I updated the preference keys per my above comment.